### PR TITLE
Client password reset

### PR DESCRIPTION
**Pull request by evie-adenoyin**

### CODE REVIEW

File: src/pages/Customer/Account/AccountDetails/AccountDetails.jsx [hunk 1]
```diff
@@ -1,98 +1,147 @@
...
+    try {
+      await customerUpdateDetails({
+        credentials: values,
+      }).unwrap()
+      saveToLocalStorage('userInfo', {
+        ...userInfo,
+        ...values,
+      })
+    } catch (err) {
+      console.log(err)
```
Comment (slimsolz) [inline src/pages/Customer/Account/AccountDetails/AccountDetails.jsx:41]: remember to remove this in the next PR

File: src/pages/Customer/Account/AccountPassword/AccountPassword.jsx [hunk 2]
```diff
@@ -4,12 +4,21 @@ import FormButton from '../../../../components/Auth/Forms/Widgets/FormButton'
...
+  const [customerChangePassword, { isLoading }] =
+    useCustomerChangePasswordMutation()
+  const onFinish = async (values) => {
     console.log('Finish:', values)
+    try {
+      await customerChangePassword({
+        credentials: values,
+      }).unwrap()
+    } catch (err) {
+      console.log(err)
```
Comment (slimsolz) [inline src/pages/Customer/Account/AccountPassword/AccountPassword.jsx:20]: same here

Comment (evie-adenoyin) [reply src/pages/Customer/Account/AccountPassword/AccountPassword.jsx:20]: I am aware of all of them.. i will remove all of them in my next PR. Thank you

**Review by slimsolz — APPROVED** [review]
LGTM 

remove the logs and handle the errors in the next PR

**Comment by slimsolz** [comment]
Resolve conflicts

**Pull request merged**